doc/INTEGRATION.md

```diff
@@ -0,0 +1,89 @@
+# Axelar Amplifier Integration Guide
+
+This document provides an overview of the contracts on Axelar Amplifier protocol and outlines the steps required to integrate a chain. The document only covers the Amplifier contracts; a comprehensive list on other required components for the integration of a chain into Amplifier can be found in the [Axelar Amplifier Public Docs](https://docs.axelar.dev/dev/amplifier/chain-integration/introduction/).
+
+---
+
+## Amplifier Contracts Overview
+
+To connect your chain to the Axelar network via the Interchain Amplifier, you will need to deploy or configure the following three contracts on Amplifier, and the configuration of `ampd`, as outlined below:
+
+### Chain-Specific Amplifier Contracts
+
+**Gateway**  
+
+A contract on Amplifier that will forward incoming messages to the chain's correspoding verifier contract to check the validity of a transaction and forwards verified transactions onto the Amplifier router. It also receives messages from other chains to be processed by the multisig prover.
+Most integrators will begin by using or customizing this Gateway Contract.
+[Gateway Documentation](../doc/src/contracts/gateway.md)
+
+**Voting Verifier**  
+
+A contract on Amplifier protocol that verifies the validity of transactions on your chain. Most integrators will begin by using or customizing a Voting Verifier or can be customized to your needs for cases such as verification via ZK Proof.
+
+A Voting Verifier must be supported by a Verifier Set will need to support the chain and vote on the truth of source chain transactions. 
+
+[Voting Verifier Documentation](../doc/src/contracts/voting_verifier.md)
+
+**Multisig Prover**  
+
+The prover contract is responsible for transforming gateway messages into a payload that is ready to be sent to the destination gateway.
+
+[Multisig Prover Documentation](../doc/src/contracts/multisig_prover.md)
+
+
+### ampd
+
+Ampd is the off chain daemon responsible for voting and signing within the amplifier protocol. The daemon listens to the Axelar blockchain for specific events, connects to external blockchains via JSON-RPC, and uses [tofnd](https://github.com/axelarnetwork/tofnd) for signing transactions and batches of messages
```

Comment:
should be more generic, voting and signing are specific to the verification and proof method we use